src/tailwind.css

```
@@ -228,6 +228,10 @@
   image-rendering: pixelated;
 }
 
+@utility grid-full {
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)